pkg/controller/job/job_controller.go

```diff
@@ -1951,6 +1945,24 @@ func countReadyPods(pods []*v1.Pod) int32 {
 	return cnt
 }
 
+// Returns the value to be set as the "terminating" field in status
```

Comment:
nit: Golang's docs convention is to start with the name of the element (source https://go.dev/blog/godoc)
```
// terminatingFieldValue returns the value to be set as the "terminating" field in status
```

Reply:
Thanks, cleaned up